src/monocle_apptrace/exporters/azure/azure_exporter.py

```diff
@@ -0,0 +1,117 @@
+import os
```

Comment:
the name of the this file can be just 'blob_exporter.py' or 'az_blob_exporter.py
'